invenio_vocabularies/contrib/funders/facets.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Vocabulary awards facets and labels."""
+
+from ...services.facets import VocabularyLabels, lazy_get_label
+
+
+class FundersLabels(VocabularyLabels):
+    """Fetching of vocabulary labels for facets."""
+
+    def __init__(self, vocabulary, cache=False, service_name=None):
+        """Initialize the labels."""
+        super().__init__(
+            vocabulary, cache, service_name="funders-service", id_field="pid"
```

Comment:
The `service_name` is needed to avoid using always the generic vocbulary service. However, this means that we need to standarize the way we register services. Currently we are using:
- <vocabulary_name>-service when in `invenio-vocabularies`. For example: `awards-service`, `names-service`.
- rdm-<vocabulary_name> when in `invenio-rdm-records`. For example: `rdm-awards`, `rdm-names`.

At the moment this means that labelling will fail when integrated.

Reply:
I agree... Would it make sense to only configure the facets in `rdm-records` maybe? We can still test in this module (with the "mock" `awards-service` configured).

Reply:
This actually make sense, we do register the facets in rdm-records for bib records (instead of drafts- or records- resources).